iOS/src/toga_iOS/screen.py

```diff
@@ -0,0 +1,28 @@
+from toga.screen import Screen as ScreenInterface
+
+
+class Screen:
+    _instances = {}
+
+    def __new__(cls, native):
+        if native in cls._instances:
+            return cls._instances[native]
+        else:
+            instance = super().__new__(cls)
+            instance.interface = ScreenInterface(_impl=instance)
+            instance.native = native
+            cls._instances[native] = instance
+            return instance
+
+    def get_name(self):
+        # Return a dummy name as UIScreen object has no name related attributes.
+        return "iOS Screen"
+
+    def get_origin(self):
+        return (0, 0)
+
+    def get_size(self):
+        return int(self.native.bounds.size.width), int(self.native.bounds.size.height)
+
+    def get_image_data(self):
+        self.interface.factory.not_implemented("Screen.get_image_data()")
```

Comment:
The implementation for this should be almost identical to the `Window.as_image` implementation, except that the cropping portion isn't required.

Reply:
Done. Thanks. Btw, nice rant on `Window.as_image` :-)